src/common/regions/utils.ts

```diff
@@ -56,3 +60,77 @@ export function getAutocompleteRegions(region?: Region) {
     return [];
   }
 }
+
+function countyIncludesZip(county: County, zipCode: string): boolean {
+  return county.zipCodes.includes(zipCode);
+}
+
+export function getStateRegionFromStateCode(
+  stateCode: string,
+): Region | undefined {
+  const regionFromStateCode = find(
+    regions.states,
+    (region: State) => region.stateCode === stateCode,
+  );
+  return regionFromStateCode;
+}
+
+export function getCountyRegionFromZipCode(
+  zipCode: string,
+): Region | undefined {
+  const countyFromZip = find(regions.counties, (region: County) =>
+    countyIncludesZip(region, zipCode),
+  );
+  return countyFromZip;
+}
+
+// TODO (Chelsi): fix the any
+export function getMetroRegionFromZipCode(zipCode: string): any {
+  const metroFromZip = find(regions.metroAreas, (region: MetroArea) =>
+    find(region.counties, (region: County) =>
+      countyIncludesZip(region, zipCode),
+    ),
+  );
+  return metroFromZip;
+}
```

Comment:
Consider finding the county first using your function from above, and then use that to find the metro area.